src/client/sandbox/code-instrumentation/location/wrapper.js

```diff
@@ -41,15 +41,17 @@ export default class LocationWrapper extends EventEmitter {
             isForm:   parsedResourceType.isForm
         });
         const getHref        = () => {
+            /*eslint-disable no-restricted-properties*/
             if (window !== window.top && window.location.href === 'about:blank')
                 return 'about:blank';
+            /*eslint-enable no-restricted-properties*/
 
             const locationUrl    = getDestLocation();
             const resolveElement = urlResolver.getResolverElement(window.document);
 
-            resolveElement.href = locationUrl;
+            nativeMethods.anchorHrefSetter.call(resolveElement, locationUrl);
 
-            return ensureTrailingSlash(resolveElement.href, locationUrl);
+            return ensureTrailingSlash(nativeMethods.anchorHrefGetter.call(resolveElement), locationUrl);
```

Comment:
Move `nativeMethods.anchorHrefGetter.call(resolveElement)` to a separate variable